api/SVGFECompositeElement.json

```diff
@@ -329,6 +353,54 @@
           }
         }
       },
+      "lighterForError": {
+        "__compat": {
+          "description": "<code>operator</code> value: <code>lighter</code>",
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": "≤18"
+            },
+            "firefox": {
+              "version_added": "86"
+            },
+            "firefox_android": {
+              "version_added": "86"
+            },
+            "ie": {
```

Comment:
Note, IE doesn't work support lighter for feComposite, so set false in that and here too (verified by testing).